src/app/sessions/components/sessions-propositions-table/sessions-propositions-table.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, EventEmitter, Input, OnChanges, Output } from '@angular/core';
+import { MatTableDataSource } from '@angular/material';
+import { SessionPropositionView } from '../../models/session-proposition-view.model';
+
+@Component({
+    selector: 'app-sessions-propositions-table',
+    templateUrl: './sessions-propositions-table.component.html',
+    styleUrls: ['./sessions-propositions-table.component.scss']
+})
+export class SessionsPropositionsTableComponent implements OnChanges {
+
+    @Input() sessionPropositions: SessionPropositionView[];
+    @Output() onSessionCreate = new EventEmitter<SessionPropositionView>();
+
+    displayedColumns = [
+        'date',
+        'startTime',
+        'endTime',
+        'availibility',
+        'judge',
+        'room',
+        'actions'
+    ];
+    dataSource: MatTableDataSource<SessionPropositionView>;
+    tableVisible: boolean;
+
+    constructor() {
+    }
+
+    ngOnChanges() {
+        if (this.sessionPropositions) {
+            this.tableVisible = true;
+            this.dataSource = new MatTableDataSource(this.sessionPropositions);
+        }
+    }
+
+    createSessionClicked(element: SessionPropositionView) {
```

Comment:
The name 'createSessionClicked' suggests as if it was an event however it's just a function triggered upon button click. I'd prefer 'createSession' (Sorry if being too picky :( )